ftps/1.6/modules/ROOT/pages/index.adoc

```diff
@@ -128,6 +128,29 @@ You can use Transport Layer Security (TLS) and configure FTPS by providing a key
 * For more on SSL configuration in Mule, see xref:mule-runtime::tls-configuration.adoc[About TLS Configuration].
 * For more on other functionality provided by this connector, see xref:ftp-connector::index.adoc[FTP Connector Documentation Reference].
 
+== Configure Java JDK for Session Reuse
+
+To reuse sessions using FTPS you must disable the Extended Master Secret extension for compatibility (JDK-8192045), which by default is set to true. The extension protects users from vulnerabilities by using the same session on different groups of clients and servers, which contradicts the demand to use the same session for control and data connections. Extended Master Secret extension activated, enables session reuse only with an endpoint verification, which is implemented for HTTPS and LDAP only.
+
+To disable the Extended Master Secret extension:
+
+* If you deploy your Mule application to a Standalone Mule runtime engine, add the following argument to the JVM execution:
```

Comment:
```suggestion
* If you deploy your Mule application to a standalone Mule runtime engine, add the following argument to the JVM execution:
```

Reply:
Another way to present these two scenarios is as follows:

"To disable the Extended Master Secret extension when working in a standalone Mule environment, add the following...:

"To disable the extension when working in Anypoint Studio, follow these steps: